spec/u3d/iniparser_spec.rb

```diff
@@ -82,5 +82,36 @@
         end
       end
     end
+
+    describe '.create_linux_ini', focuss: true do
+      context 'existing ini file' do
+        it 'does not rewrite the file' do
+          path = %r{\/.u3d\/ini_files\/unity-1.2.3f4-linux.ini}
+
+          allow(U3dCore::Helper).to receive(:operating_system) { :linux }
```

Comment:
You can use shortcut `on_linux` from the setups

Reply:
👍